obspy/clients/fdsn/tests/test_client.py

```diff
@@ -668,6 +669,7 @@ def test_str_method(self):
         expected = normalize_version_number(expected)
         self.assertEqual(got, expected, failmsg(got, expected))
 
+    @unittest.expectedFailure
```

Comment:
Does anyone know if IRIS has changed how they respond to `minimumlength`?

Reply:
Maybe @chad-iris or @nick-iris know or could find out for us?